src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1307,7 +1322,7 @@ public JSONObject getDebugState(boolean full, String endpointId)
     {
         JSONObject debugState = new JSONObject();
         debugState.put("id", id);
-        debugState.put("name", name == null ? null : name.toString());
+        debugState.put("name", conferenceName == null ? null : conferenceName);
```

Comment:
That's equivalent to just `conferenceName`. Did you want the string "null"?

Reply:
Ha, oops.  I just wanted to recreate the old logic, which looks like would be just using `conferenceName`

Reply:
Fixed, thanks.